analyses/cms-open-data-ttbar/statistical_interference/agc_sample.py

```diff
@@ -0,0 +1,310 @@
+import ROOT
+
+class AGC_Sample(ROOT.RooStats.HistFactory.Sample):
```

Comment:
```suggestion
class AGCSample(ROOT.RooStats.HistFactory.Sample):
```
To make the linter happy